### PR TITLE
F∅ Initial Deployment

### PR DESCRIPTION
**Pull request by loongy**
**Description**

Deployment to the F∅ Testnet. This PR will be deployed to Falcon Testnet, and if the final stage of testing works this PR will be merged.

**Motivation**

F∅ is deployed from the `master` branch.

**Unresolved Issues**

The linter is still disabled as comments and style continue to be propagated throughout the project.

### CODE REVIEW

**Comment by noahingham** [comment]
`Join` returns an error - should it be checked here?

https://github.com/republicprotocol/republic-go/blob/ee4cb88af2b9c3672f44297d29f23caa5e14adbd/ome/matcher.go#L106-L108

**Comment by loongy** [comment]
Yes it should. Fix for that has been pushed.

File: cmd/analysis/renLedger/main.go [hunk 1]
```diff
@@ -56,15 +57,15 @@ func main() {
 			log.Fatal(err)
 		}
 
-		log.Printf("%d | %v | %v ", i, hexID, status)
+		log.Printf("%d | %v | %v | %v", i, hexID, status, base64.StdEncoding.EncodeToString(orderID[:]))
 	}
 
-	log.Printf("Open Orders : %v, Confirmed Orders: %v", openCounts, confirmedCount)
+	log.Printf("Open Orders : %v, Confirmed Orders: %v, base64 : ", openCounts, confirmedCount)
```
Comment (tok-kkk) [inline cmd/analysis/renLedger/main.go]: remove the `base64`

File: grpc/grpc.proto [hunk 2]
```diff
@@ -53,16 +53,18 @@ message OpenOrderResponse {
 }
 
 message EncryptedOrderFragment {
-    bytes       orderId     = 1;
-	OrderType   orderType   = 2;
-	OrderParity orderParity = 3;
-    int64       orderExpiry = 4;
+    bytes           orderId         = 1;
+	OrderType       orderType       = 2;
```
Comment (tok-kkk) [inline grpc/grpc.proto]: failure of vs code

**Pull request merged**